Gemfile

```diff
@@ -38,6 +38,7 @@ group :development, :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
+  gem 'rubocop'
```

Comment:
Could you try 'bixby' instead?  Hopefully it would give us 💚 with cops already configured for the hyrax generated code.